server/public/styles.css

```diff
@@ -23,6 +23,16 @@ h6 {
   height: 100%;
 }
 
+.fit-summary {
+  object-fit: contain;
+  width: 100%;
+  height: 250px;
+}
+
 .pointer {
   cursor: pointer;
 }
+
+.summary>* {
+  width: 100%;
+}
```

Comment:
Uh, no. Please try to be more intentional with your CSS. What are you trying to do here?